src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs

```diff
@@ -27,6 +28,7 @@ public class TemplateController : Controller
         private readonly IStringLocalizer S;
         private readonly IHtmlLocalizer H;
         private readonly dynamic New;
+        private readonly IHttpContextAccessor _hca;
```

Comment:
Controllerben a `HttpContext` elérhető az ilyen nevű property-ből is.

Reply:
Hogy érted? TemplateController-ben eleve úgy érem el hogy _hca.HttpContext.Request

Reply:
Nincs szükség beinjektálni az `IHttpContextAccessor`-t, mert a `HttpContext` property-ből egy controllerben anélkül is elérhető.

Reply:
Ahha, szuper